src/app/pages/RewardPage/components/HistoryTable/index.tsx

```diff
@@ -1,85 +1,48 @@
-import axios from 'axios';
-import React, { useCallback, useEffect, useState } from 'react';
-
-import { Pagination } from 'app/components/Pagination';
-import { useAccount, useBlockSync } from 'app/hooks/useAccount';
-import { backendUrl, currentChainId } from 'utils/classifiers';
-
-import { RewardEvent, TableBody } from './TableBody';
+import React, { useCallback, useMemo, useState } from 'react';
+import { TableBody } from './TableBody';
 import { TableHeader } from './TableHeader';
-import { RewardTabType } from '../../types';
-
-const pageSize = 6;
-
-const getHistoryEndpoint = (activeTab: RewardTabType) => {
-  switch (activeTab) {
-    case RewardTabType.REWARD_SOV:
-      return 'rewardsNew';
-    case RewardTabType.LIQUID_SOV:
-      return 'liquidSov';
-    case RewardTabType.FEES_EARNED:
-      return 'feesEarned';
-    default:
-      return 'rewardsNew';
-  }
-};
+import { useGetUserRewardsEarnedHistory } from '../../hooks/useGetUserRewardsEarnedHistory';
+import { RewardPagination } from '../RewardPaginaton';
 
-interface IHistoryTableProps {
-  activeTab: RewardTabType;
-}
+const PAGE_SIZE = 5;
```

Comment:
This is definitely a good approach but it will be repetitive in the future as we're gonna use it everywhere. Pls, rename it to `DEFAULT_PAGE_SIZE` and move it to `utils/classifiers.ts`.

Reply:
yes, right. I'm planing to move this pagination in common component and will replace it in my PRs once one of them will be merged into dev